QuizApp/QuizApp/UI/Screens/Quiz/QuizViewModel.swift

```diff
@@ -39,10 +68,12 @@ class QuizViewModel {
             return .musicColor
         case .geography:
             return .geographyColor
+        case .all:
+            return .allColor
         }
     }
 
-    private func getCategoryModel(from type: CategoryType) -> CategoryModel {
+    private func getCategoryModel(from type: CategoryType) -> CategoryModel? {
```

Comment:
You can use `CategoryModel(rawValue: type.rawValue)`. That constructor returns an optional value so it should return nil for `.all`.